developer-guide.md

```diff
@@ -103,21 +103,23 @@ Yes, you can technically request a token for an origin that you don't own. Howev
     Please see [How do I renew a token that is about to expire/has expired?](#renew) for how to renew your tokens 
     to continue participating in the trial.
 
+### 12. Do I need to provide feedback for the trial if I request a token?
+
+As of February 12, 2023, submitting feedback for a trial is optional, but encouraged. Over the course of your participation in the trial, you will receive some notifications by email prompting for your feedback on the feature, along with instructions on how to do so.
+
 <a name="renew"></a>
-### 12. How do I renew a token that is about to expire/has expired? 
+### 13. How do I renew a token that is about to expire/has expired? 
 
 - You should receive a reminder email to renew the token before it expires. That email includes a 
     link to the registration page in the [developer console](https://developers.chrome.com/origintrials/#/trials/active).
     You can also go directly to the page in the console, by finding the trial in the list on
     [My Registrations](https://developers.chrome.com/origintrials/#/trials/my).
-- On the registration page, you may need to provide feedback before you can renew. Use the 
-    Feedback button to complete a survey to provide feedback.
-- On the registration page, the Renew button will be available when feedback has been provided. 
-    Use the Renew button to generate a new token.
-- Feedback is required to renew tokens. If you have multiple origins registered for a trial,
-    we'll only ask you for once per period, rather than for every origin.
+- On the registration page, use the Renew button to generate a new token.
+- Tokens are set to expire only at the end of the registered origin trial. However, in some cases,
```

Comment:
This point is already covered in the previous question (11), and can be removed. If desired, you could revise the text of 11, but I don't think it should be added here.